packages/model/addon/-private/model.js

```diff
@@ -2157,7 +2163,7 @@ Model.reopenClass({
       callback.call(binding, name, meta);
     });
   },
-
+  
   /**
```

Comment:
☝️

Reply:
@snewcomer I'm assuming you are pointing out the example for `eachAttribute`.  I believe this would be the native class equivalent??

```js
   import Model, { attr } from '@ember-data/model';

   const person = new Model({
      @attr('string') firstName;
      @attr('string') lastName;
      @attr('date') birthday;
    })

   person.eachAttribute(function(name, meta) {
      console.log(name, meta);
    });
```

Reply:
updated